File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        
        arr = []
        j = 0
        for i in range(0,2*n,2):
            arr.insert(i,nums[j])
            arr.insert(i+1,nums[n+j])
            j = j + 1
        return arr

File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] arr = new int[nums.length];
        for (int i = 0; i < n;i++){
            arr[2*i] = nums[i];
            arr[2*i+1] = nums[n+i];
        }
        return arr;
    }
}